src/components/description-list/_description-list.scss

```diff
@@ -7,26 +7,39 @@
         clear: both;
         float: left;
         font-weight: $font-weight-bold;
-
-        &:not(:first-child) {
-            margin-top: 0.5rem;
-        }
+        word-break: break-word;
```

Comment:
We can remove this, its deprecated and allows the breaking of words which is what the default does anyway